Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,176 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+}
+
+public extension View {
+
+    /// Elevates the view to make it more prominent.
+    func elevation(_ level: Elevation) -> some View {
+        elevation(level, prerender: true)
+    }
+}
+
+extension View {
+
+    func elevation(_ level: Elevation, prerender: Bool) -> some View {
+        modifier(ElevationModifier(level: level, isPrerendered: prerender))
+    }
+}
+
+struct ElevationModifier: ViewModifier {
+
+    @Environment(\.isElevationEnabled) var isElevationEnabled
+    let level: Elevation
+    let isPrerendered: Bool
+    let shadowColor = Color(red: 79 / 255, green: 94 / 255, blue: 113 / 255)
+
+    func body(content: Content) -> some View {
+        if isElevationEnabled {
+            if isPrerendered {
+                content
+                    .background(prerenderedShadow(content: content))
+            } else {
+                content
+                    .background(dynamicShadow(content: content))
+            }
+        } else {
+            content
+        }
+    }
+
+    @ViewBuilder func prerenderedShadow(content: Content) -> some View {
+        switch level {
+            case .level1:
+                content
+                    .shadow(color: shadowColor.opacity(0.28), radius: 2.6, y: 1.7)
+                    .padding(.small)
+                    .drawingGroup(colorMode: .extendedLinear)
+            case .level2:
+                content
+                    .shadow(color: shadowColor.opacity(0.18), radius: 4, y: 4)
+                    .shadow(color: shadowColor.opacity(0.12), radius: 8, y: 8)
+                    .padding(.medium)
+                    .drawingGroup(colorMode: .extendedLinear)
+            case .level3:
```

Comment:
Suggestions for lvl3+4

```
case .level3:
                content
                    .shadow(color: shadowColor.opacity(0.28), radius: 2.8, y: 2.4)
                    .shadow(color: shadowColor.opacity(0.16), radius: 12, y: 12)
                    .padding(.xLarge)
                    .drawingGroup(colorMode: .extendedLinear)
            case .level4:
                content
                    .shadow(color: shadowColor.opacity(0.3), radius: 3.2, y: 2.7)
                    .shadow(color: shadowColor.opacity(0.19), radius: 18, y: 18)
                    .padding(.xxLarge)
                    .drawingGroup(colorMode: .extendedLinear)
```

<img width="983" alt="image" src="https://user-images.githubusercontent.com/35844477/175265338-b3530eb0-bc3d-478b-a5fa-8899342bb752.png">